packages/angular/src/generators/ngrx/files/__directory__/__fileName__.facade.ts__tmpl__

```diff
@@ -7,6 +7,8 @@ import * as <%= className %>Selectors from './<%= fileName %>.selectors';
 
 @Injectable()
 export class <%= className %>Facade {
+  private readonly store = inject(Store);
```

Comment:
Is this the new recommendation?

Reply:
No new recommendation has emerged yet, however, this allows for TC39 compliant code and also ensures that the NgRx documentation remains valid for users.